README.md

```diff
@@ -1 +1,31 @@
-# angular-what-is-angular-readme
+# What is Angular and why does it exist?
+
+## Overview
+
+AngularJS (commonly referred to as just "Angular") is an open-source framework released in 2009, being mainly maintained by Google as well as a massive community of developers.
+
+It aims to solve the problems faced when creating single page applications, implementing a MVVM pattern to separate presentation and business logic.
+
+## Objectives
+
+- Describe what Angular's role in the web platform is
+- Describe the issues Angular looks to solve
+- Describe how Angular thinks in terms of Web Components to create encapsulation and reusable components
+
+## Traditional Rendering
+
+Before mainstream JavaScript usage, everything on the client was determined by the server. Routing and views were defined in backend code, meaning view logic was frequently mixed in with business logic, providing the end-user with a poor user experience.
+
+This had its disadvantages, as data displayed on the page could only be updated when the user refreshes. This also meant no smooth transitions between pages.
+
+## The role of Angular
```

Comment:
Here you could start to introduce how Angular came to be created, who developed it, some sites that are built with Angular, etc. By the end of this README, the student should be really excited about learning Angular. 

For example, this is how we start a README on "What are Web Applications?"

> How does Twitter have different pages for each of its 300 million users? How can AirBnB support over 40 million guests with over 1.5 million listings worldwide? All of this is possible because of web applications.- for example this is how we start our first lesson on web applications

Here's another example from the Intro Rails README:

> Welcome to the world of Ruby on Rails development! With over a decade of open source contributions, Rails has evolved into one of the most powerful web application frameworks available. Before we can start building applications it is important to first understand what the Ruby on Rails framework is.. and what it's not.